### ISSUE TITLE
Softmax

### ISSUE THREAD

**Issue by lhy2749** [iworldtong/text_gcn.pytorch#4]
Why not use softmax for gcnlayer2?
